.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 30
```

Comment:
Does 30 seem...a bit aggressive? Do we really have that many issues/prs that we can't do this sort of triage manually and not risking closing relevant issues?

Reply:
Oh I didn't realize all the labels in `exempt-issue-labels`. I think this is probably fine.

Reply:
Right - if you dont want to close the issue, then just apply a label to it. We can adjsut the list of exempt labels. This is just a "POC"